tests/keras/test_keras_model_export.py

```diff
@@ -171,6 +171,7 @@ def save(self, path, **kwargs):
     class FakeKerasModule(object):
         __name__ = "some.test.keras.module"
         __version__ = "42.42.42"
+        __package__ = ""
```

Comment:
Is this attribute used somewhere?

Reply:
Good catch! Left over from previous diff. Removed!